app/code/Magento/Captcha/Controller/Adminhtml/Refresh/Refresh.php

```diff
@@ -11,20 +11,48 @@
 class Refresh extends \Magento\Backend\App\Action
 {
     /**
+     * @var \Magento\Framework\Serialize\SerializerInterface
+     */
+    protected $serializer;
+
+    /**
+     * @var \Magento\Captcha\Helper\Data
+     */
+    protected $captchaHelper;
+
+    /**
+     * Refresh constructor.
+     * @param \Magento\Backend\App\Action\Context $context
+     * @param \Magento\Captcha\Helper\Data $captchaHelper
+     * @param \Magento\Framework\Serialize\SerializerInterface|null $serializer
+     * @throws \RuntimeException
+     */
+    public function __construct(
+        \Magento\Backend\App\Action\Context $context,
+        \Magento\Captcha\Helper\Data $captchaHelper,
+        \Magento\Framework\Serialize\SerializerInterface $serializer = null
+    ) {
+        parent::__construct($context);
+        $this->serializer = $serializer ?: \Magento\Framework\App\ObjectManager::getInstance()
```

Comment:
* Please use `Magento\Framework\Serialize\Serializer\Json` instead of `SerializerInterface`, since we can not guarantee that preference will remain Json in the future.
* Since you are not modifying the existing constructor but adding a new one – you can inject the dependency in a normal way.